message/validation/common_checks.go

```diff
@@ -117,10 +117,20 @@ func (mv *messageValidator) validateBeaconDuty(
 	role spectypes.RunnerRole,
 	slot phase0.Slot,
 	indices []phase0.ValidatorIndex,
+	partialMessageType *spectypes.PartialSigMsgType,
```

Comment:
I feel this could be just replaced with a boolean? and do the check outside. since you only use it to see if its a randao message. this could just be `randao bool`